src/chopper_pack.cpp

```diff
@@ -79,6 +80,9 @@ int set_up_and_parse_subparser_split(seqan3::argument_parser & parser, pack_conf
                     "If given together with --determine-num-bins, all binnings up to the chosen t_max are computed "
                     "instead of stopping when the expected query costs become worse.");
 
+    parser.add_flag(config.verbose, 'v', "verbose", "Print additional statistics and information to the "
+                    "command line.");
+
```

Comment:
Do you think that the information are valuable to the user who wants to pack their data?
Or is it rather for us to benchmark the algorithm?

If the former, you can leave the option but if its the latter, I would move this option to the end in the section `special options` (I added sections in some PR that is already merged) and call it `--output-statistics`